DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -252,8 +252,8 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="16">
-   <version ClassVersion="17" checksum="1633877368"/>
+  <class name="pat::PackedCandidate" ClassVersion="17">
+    <version ClassVersion="17" checksum="1257500115"/>
```

Comment:
mhh in any case I expect the checksum to be neither of the two if both PRs
are changing something... so is not just a matter of increasing 17 to 18,
right?

On Sun, Mar 15, 2015 at 6:49 PM, David Lange <notifications@github.com>
wrote:

> In DataFormats/PatCandidates/src/classes_def_objects.xml
> <https://github.com/cms-sw/cmssw/pull/8122#discussion_r26450034>:
>
> > @@ -252,8 +252,8 @@
> >     <version ClassVersion="11" checksum="1239840459"/>
> >    </class>
> >
> > -  <class name="pat::PackedCandidate" ClassVersion="16">
> > -   <version ClassVersion="17" checksum="1633877368"/>
> > +  <class name="pat::PackedCandidate" ClassVersion="17">
> > +    <version ClassVersion="17" checksum="1257500115"/>
>
> your question assumes a particular future path of the release builds.
> While likely true, lets try not to repeat mistakes of the past…
> … <#14c1e8e21ca76eb6_>
>  On Mar 15, 2015, at 6:39 PM, arizzi <notifications@github.com> wrote: In
> DataFormats/PatCandidates/src/classes_def_objects.xml: > @@ -252,8 +252,8
> @@ > <version ClassVersion="11" checksum="1239840459"/> > </class> > > -
> <class name="pat::PackedCandidate" ClassVersion="16"> > - <version
> ClassVersion="17" checksum="1633877368"/> > + <class
> name="pat::PackedCandidate" ClassVersion="17"> > + <version
> ClassVersion="17" checksum="1257500115"/> well, does it matter? I mean if
> no release exists with "17" why should we care about increasing the version
> number? — Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8122/files#r26450034>.
>